README.md

```diff
@@ -1 +1,152 @@
-# Portfolio_Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  
+  <h1><b>Portfolio Microverse</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Personal Portfolio <a name="about-project"></a>
+
+This is the Portfolio Microverse project! I've created it using HTML and CSS. It includes the implementation of the Toolbar and Headline sections as per the provided design.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.spec.whatwg.org/multipage//">HTML</a></li>
+    <li><a href="https://www.w3.org/TR/CSS/#css/">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Implemented the Toolbar and Headline sections.
+- Utilized flexbox for layout and responsiveness.
+- Styled the elements to match the design, including fonts, colors, images, text, and margins.
+- Added button interactions such as hover effects.
+- Ensured responsiveness using responsive values in CSS rules, like percentages instead of pixels.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project, you need:
+
+1. A web browser.
+2. A code editor.
+3. Git.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run the following command:
+
+```sh
+cd your-folder
+git clone git@github.com:IsaMarvin/Portfolio_Microverse.git
+```
+
+### Install
+
+Install this project with:
+
+Run command:
+
+- cd your-folder
+  
+  ``` npm install ```
+
+
+
+### Usage
+
+To run the project, execute the following command:
+
+Open index.html using live server extension.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 IsaMarvin
+
+- GitHub: [IsaMarvin](https://github.com/IsaMarvin)
+- LinkedIn: [IsaMarvin](https://www.linkedin.com/in/isamarvin/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [OPTIONAL] The issues link here is redirecting to a **_page not found_** error ⚠ This can be fixed by using the absolute path for the issues page instead  of the relative page. You can consider 👇

  `[issues page](https://github.com/IsaMarvin/Portfolio_Microverse/issues)`